client/src/kit/link.tsx

```diff
@@ -3,6 +3,16 @@ import { Link as RouterLink } from "react-router-dom";
 
 export const Anchor = styled("a")`
   color: var(--fg-a);
+  font-weight: var(--fw-b);
+  font-size: var(--fs-m);
+  text-decoration: none;
+  display: inline-block;
+  padding: 0 0 11px;
```

Comment:
display and padding don't make sense here for me. give a reason for these changes or revert them